python/tchannel/messages/call_continue.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import absolute_import
+
+from . import common, ChecksumType
+from .base import BaseMessage
+from .common import FlagsType
+
+
+class CallContinueMessage(BaseMessage):
+    """Represent a continuation of a call request (across multiple frames)."""
+    max_args_num = 3
+
+    __slots__ = (
+        'flags',
+        'checksum',
+        'args',
+    )
+
+    def __init__(
+        self,
+        flags=0,
+        checksum=None,
+        args=None,
+    ):
+        self.flags = flags
+        if checksum is not None:
+            checksum = common.ChecksumType.standardize(checksum)
+        self.checksum = checksum or \
+            (common.ChecksumType.none, None)
+
+        self.args = args or []
+
+    def encode(self):
+        self.args = list(map(
+            lambda arg: common.encode_str(arg),
+            self.args))
+
+    def decode(self):
+        self.args = list(map(
+            lambda arg: common.decode_str(arg),
+            self.args))
+
+    def get_meta_size(self):
+        size = 0
+        size += 1       # flags: 1
+        size += 1       # csumtype: 1
+        size += 1 if self.checksum[0] != ChecksumType.none else 0
+        return size
+
+    def fragment(self, space_left, fragment_msg):
+        """Streaming Message got fragmented based on
+        payload size. All the data within space_left
+        will be kept. All the rest will be shifted to
+        next fragment message.
+
+        :param space_left: space left for current frame
+        :param fragment_msg: the type is either
+        CallRequestMessage or CallResponseMessage
```

Comment:
For multi-line `:param:` entries, move the description to the next line and indent 4 spaces.

    :param fragment_msg:
        Multi-line description
        goes here